test/spec/Radicle/Tests.hs

```diff
@@ -475,6 +475,16 @@ test_eval =
         runPureCode "(show (dict 'a 1))" @?= Right (String "{a 1}")
         runPureCode "(show (fn [x] x))" @?= Right (String "(fn [x] x)")
 
+    , testCase "'show-unbounds' works" $ do
+        runPureCode "(show-unbound {:b 2 :a 1})" @?= Right (String "{:a 1 :b 2}")
+        runPureCode "(show-unbound {:bar \"bar bar bar bar bar bar bar bar bar bar bar bar bar bar bar\" :foo \"foo foo foo foo foo\"})"
+          @?= Right (String "{:bar \"bar bar bar bar bar bar bar bar bar bar bar bar bar bar bar\" :foo \"foo foo foo foo foo\"}")
+
+    , testProperty "'show-unbounds does not add any whitespace characters'" $ \(NoSpaceValue val :: NoSpaceValue) -> do
+        let res = runPureCode $ toS [i|(show-unbound (quote #{renderPrettyUnbounded val}))|]
```

Comment:
Is the quote needed?

Reply:
Should only be around the function name

Reply:
Not sure I understand.

Right now it seems like some pretty-printing is not being tested because of quote. E.g., lambdas.

Reply:
Sorry, misread your message (which pointed me to a different typo)

The quote is needed as otherwise on show/show-unformatted unknown identifiers (that are arbitrarily created) lead to errors. Quoting prevents that. I don't see what the problem with lambdas is, at least from testing in repl.

Reply:
```
rad> (type (quote (fn [] 3)))
:list
rad> (type (fn [] 3))
:function
```
So you're never giving `show-unbound` an argument that's a function, as far as I can tell.

You can just make the test assert that *if* the result is not an error, it satisfies your property (or else discard errors).